DocumentDataAPI/DocumentDataAPI/Data/Repositories/NpgDocumentRepository.cs

```diff
@@ -150,4 +150,18 @@ public async Task<int> GetTotalDocumentCount()
         using IDbConnection con = _connectionFactory.CreateConnection();
         return await con.QuerySingleAsync<int>($"select count({DocumentMap.Id}) from documents");
     }
+
+    public async Task<IEnumerable<string>> GetAuthors()
+    {
+        _logger.LogDebug("Retrieving all authors from database");
+        using IDbConnection con = _connectionFactory.CreateConnection();
+        return await con.QueryAsync<string>($"select distinct {DocumentMap.Author} from documents");
+    }
+
+    public async Task<IEnumerable<CategoryModel>> GetCategories()
+    {
+        _logger.LogDebug("Retrieving all categories from database");
+        using IDbConnection con = _connectionFactory.CreateConnection();
+        return await con.QueryAsync<CategoryModel>($"select distinct {DocumentMap.CategoryId} from documents");
+    }
```

Comment:
Skal høre til `NpgCategoryRepository`.
Derudover er querien problematisk idet du kun henter kategori-IDer ned som findes på dokumenter (i stedet for alle), og du får ikke navnet på kategorierne med, så det vil være null i de resulterende `CategoryModel` - og vi vil jo gerne vise navnet i fronten, samt bruge IDet til søgningen. 
Så det ville i stedet f.eks. være `select * from categories`